<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>opcja6</class>
 <widget class="QDialog" name="opcja6">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="obiekty">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edytuj</string>
   </property>
  </widget>
  <widget class="QFrame" name="ramka">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QTextEdit" name="textEdit_3"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Numer</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nazwa</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="textEdit_2"/>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QTextEdit" name="textEdit_4"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Typ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>opcja6</receiver>
   <slot>edytuj()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>obiekty</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>opcja6</receiver>
   <slot>doedycji()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doedycji()</slot>
  <slot>edytuj()</slot>
 </slots>
</ui>
